webpack/assets/javascripts/foreman_ssh_keys.js

```diff
@@ -0,0 +1,12 @@
+import $ from 'jquery';
+
+export function autofillSshKeyName() {
```

Comment:
two comments:
1. missing tests for functions
2. maybe its just me, but I would love seeing it implemented via react instead ;)